pkg/networkservice/core/trace/common_test.go

```diff
@@ -129,30 +132,142 @@ func TestTraceOutput(t *testing.T) {
+			"\x1b[31m [ERRO] [id:conn-1] [name:TestTraceOutput] [type:NetworkService] \x1b[0m\t/root/go/pkg/mod/github.com/networkservicemesh/sdk@v0.5.1-0.20210929180427-ec235de055f1/pkg/networkservice/core/trace/client.go:106\n" +
+			"\x1b[31m [ERRO] [id:conn-1] [name:TestTraceOutput] [type:NetworkService] \x1b[0mgithub.com/networkservicemesh/sdk/pkg/networkservice/core/next.(*nextClient).Close\n" +
+			"\x1b[31m [ERRO] [id:conn-1] [name:TestTraceOutput] [type:NetworkService] \x1b[0m\t/root/go/pkg/mod/github.com/networkservicemesh/sdk@v0.5.1-0.20210929180427-ec235de055f1/pkg/networkservice/core/next/client.go:65\n" +
+			"\x1b[31m [ERRO] [id:conn-1] [name:TestTraceOutput] [type:NetworkService] \x1b[0m(1.3)   Error returned from api/pkg/api/networkservice/networkServiceClient.Close\n" +
+			"\x1b[31m [ERRO] [id:conn-1] [name:TestTraceOutput] [type:NetworkService] \x1b[0mgithub.com/networkservicemesh/sdk/pkg/networkservice/core/trace.(*beginTraceClient).Close\n" +
+			"\x1b[31m [ERRO] [id:conn-1] [name:TestTraceOutput] [type:NetworkService] \x1b[0m\t/root/go/pkg/mod/github.com/networkservicemesh/sdk@v0.5.1-0.20210929180427-ec235de055f1/pkg/networkservice/core/trace/client.go:85\n" +
+			"\x1b[31m [ERRO] [id:conn-1] [name:TestTraceOutput] [type:NetworkService] \x1b[0mgithub.com/networkservicemesh/sdk/pkg/networkservice/core/next.(*nextClient).Close\n" +
+			"\x1b[31m [ERRO] [id:conn-1] [name:TestTraceOutput] [type:NetworkService] \x1b[0m\t/root/go/pkg/mod/github.com/networkservicemesh/sdk@v0.5.1-0.20210929180427-ec235de055f1/pkg/networkservice/core/next/client.go:65\n" +
+			"\x1b[31m [ERRO] [id:conn-1] [name:TestTraceOutput] [type:NetworkService] \x1b[0mgithub.com/networkservicemesh/sdk/pkg/networkservice/core/next.(*nextClient).Close\n" +
+			"\x1b[31m [ERRO] [id:conn-1] [name:TestTraceOutput] [type:NetworkService] \x1b[0m\t/root/go/pkg/mod/github.com/networkservicemesh/sdk@v0.5.1-0.20210929180427-ec235de055f1/pkg/networkservice/core/next/client.go:65\n" +
+			"\x1b[31m [ERRO] [id:conn-1] [name:TestTraceOutput] [type:NetworkService] \x1b[0mgithub.com/networkservicemesh/sdk/pkg/networkservice/core/trace.(*endTraceClient).Close\n" +
+			"\x1b[31m [ERRO] [id:conn-1] [name:TestTraceOutput] [type:NetworkService] \x1b[0m\t/root/go/pkg/mod/github.com/networkservicemesh/sdk@v0.5.1-0.20210929180427-ec235de055f1/pkg/networkservice/core/trace/client.go:106\n" +
+			"\x1b[31m [ERRO] [id:conn-1] [name:TestTraceOutput] [type:NetworkService] \x1b[0mgithub.com/networkservicemesh/sdk/pkg/networkservice/core/next.(*nextClient).Close\n" +
+			"\x1b[31m [ERRO] [id:conn-1] [name:TestTraceOutput] [type:NetworkService] \x1b[0m\t/root/go/pkg/mod/github.com/networkservicemesh/sdk@v0.5.1-0.20210929180427-ec235de055f1/pkg/networkservice/core/next/client.go:65\n"
+
+	// Logger created by the trace chain element uses custom formatter, which prints date and time info in each line
```

Comment:
This looks like a duplicate of lines 158-175, can you please move it to separate function?